server/src/routes/api/v1/userDataRouter.js

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+import { User } from "../../../models/index.js"
+
+const userDataRouter = new express.Router()
+
+userDataRouter.get("/:id", async (req,res) => {
+    const userID = req.params.id
+    try {
+        const queriedUser = await User.query().findById(userID)
+        const userBuildsList = await queriedUser.$relatedQuery("builds")
+        const userReviewsList = await queriedUser.$relatedQuery("reviews")
```

Comment:
you should be using a serializer right here, and then also serializing the builds and reviews